pom.xml

```diff
@@ -24,6 +24,10 @@
       <email>Sean.Roy@gmail.com</email>
       <organizationUrl>https://github.com/SeanRoy/lambduh-maven-plugin</organizationUrl>
     </developer>
+    <developer>
+      <name>Philip M. White</name>
+      <email>philip@mailworks.org</email>
+    </developer>
   </developers>
 
```

Comment:
Please move yourself to a contributors section.  I won't be adding other committers at this time.

 ```
<contributors>
  <contributor>
    <name>Philip M. White</name>
    <email>philip@mailworks.org</email>
  <contributor>
</contributors>
 ```